Backend/63/api/users/users.controller.ts

```
@@ -68,5 +85,20 @@ export const logout: RequestHandler = async (req, res, next) => {
       message: 'Bad request',
       error: (err instanceof Error && err.message) || err,
     });
+    return;
+  }
+};
+
+export const getAllUsers: RequestHandler = async (_req, res) => {
+  try {
+    const users = await User.find().select('password');
```

do u want to return only user passwords?) 
if u want to except password from user objects- then use `.select('-password')`